lane.py

```diff
@@ -1356,27 +1353,12 @@ def works(self, _db, facets=None, pagination=None, include_quality_tier=False):
                 mw.collection_id.in_(self.collection_ids)
             )
         qu = self.apply_filters(_db, qu, facets, pagination)
-        if qu:
-            qu = qu.options(
-                contains_eager(mw.license_pool),
-                # TODO: Strictly speaking, these joinedload calls are
-                # only needed by the circulation manager. This code could
-                # be moved to circulation and everyone else who uses this
-                # would be a little faster. (But right now there is no one
-                # else who uses this.)
-
-                # These speed up the process of generating acquisition links.
-                joinedload("license_pool", "delivery_mechanisms"),
-                joinedload("license_pool", "delivery_mechanisms", "delivery_mechanism"),
-                # These speed up the process of generating the open-access link
-                # for open-access works.
-                joinedload("license_pool", "delivery_mechanisms", "resource"),
-                joinedload("license_pool", "delivery_mechanisms", "resource", "representation"),
-            )
```

Comment:
Very similar code needs to run whether the objects being fetched are `MaterializedWorkWithGenre` or `Work`. I moved this code into `_modify_loading` (formerly `_lazy_load`).